python/lsst/daf/base/propertyContainer/propertySet.cc

```diff
@@ -54,7 +54,9 @@ PYBIND11_MODULE(propertySet, mod) {
             .def("__eq__",
                  [](std::type_info const& self, std::type_info const& other) { return self == other; })
             .def("__ne__",
-                 [](std::type_info const& self, std::type_info const& other) { return self != other; });
+                 [](std::type_info const& self, std::type_info const& other) { return self != other; })
+            .def("name", &std::type_info::name)
+            .def("hash_code", &std::type_info::hash_code);
```

Comment:
Not `__hash__`?

Reply:
I fear that will make it harder to find (I only added these fields for debugging). But what the heck, I'll change it. If folks really want it they can read the wrapper.